extensions-core/core/deployment/src/main/java/org/apache/camel/quarkus/core/deployment/NativeImageProcessor.java

```diff
@@ -302,14 +302,14 @@ void process(
                     org.apache.camel.main.DefaultConfigurationProperties.class,
                     org.apache.camel.main.MainConfigurationProperties.class,
                     org.apache.camel.main.HystrixConfigurationProperties.class,
-                    org.apache.camel.main.Resilience4jConfigurationProperties.class,
-                    org.apache.camel.main.RestConfigurationProperties.class));
+                    org.apache.camel.main.Resilience4jConfigurationProperties.class));
```

Comment:
Nitpick: you wanted to move Resilience4jConfigurationProperties, not RestConfigurationProperties

Reply:
damn, will fix it later to avoid another build

Reply:
Let it be!

Reply:
fixed